### PR TITLE
feat: handle formium hidden fields

### PR DESCRIPTION
**Pull request by csiyang**
# Description

### <samp>🤖 Generated by Copilot at df4602a</samp>

This pull request adds a new onboarding form page for the journeys-admin app, using Formium and next-firebase-auth. It also updates the FormiumForm component to include the user's id and email in the form submission.

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/34356604/todos/6561382989)

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Walkthrough

### <samp>🤖 Generated by Copilot at df4602a</samp>

*  Add a new page component for the onboarding form ([link](https://github.com/JesusFilm/core/pull/1933/files?diff=unified&w=0#diff-719483df49549a751a6b58ff274ddf6312157c1665900db7e79e213f88feb30eR1-R60))
* Modify the `FormiumForm` component to accept the user's id and email as props ([link](https://github.com/JesusFilm/core/pull/1933/files?diff=unified&w=0#diff-e2e419b179c8247666bfff842d0c96b4e94d397225b6830fc57c2193c972112eL42-R56))
* Change the comment style in the `Header` component from a JSDoc comment to a regular comment ([link](https://github.com/JesusFilm/core/pull/1933/files?diff=unified&w=0#diff-1372929933c5118e1d5334f60b860d2494f1c41ca55204e9866074c8d1815c77L6-R6))

### CODE REVIEW

File: apps/journeys-admin/pages/onboarding-form/index.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import { formiumClient } from '@core/shared/ui/formiumClient'
+import { FormiumForm } from '@core/shared/ui/FormiumForm'
+
+import { initAndAuthApp } from '../../src/libs/initAndAuthApp'
+
+interface OnboardingFormPageProps {
+  form: Form
+}
+
+function OnboardingFormPage({ form }: OnboardingFormPageProps): ReactElement {
```
Comment (csiyang) [inline apps/journeys-admin/pages/onboarding-form/index.tsx:20]: will remove this page after testing is compeleted

File: libs/shared/ui/src/components/FormiumForm/Header/Header.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import Box from '@mui/material/Box'
 import Typography from '@mui/material/Typography'
 import { ReactElement } from 'react'
 
-/** Contains information for the page and renders the header */
+// Contains information for the page and renders the header
```
Comment (tataihono) [inline libs/shared/ui/src/components/FormiumForm/Header/Header.tsx:6]: is this intentional? the former was a jsdoc comment and shows up in vscode for the header props file.

File: apps/journeys-admin/pages/onboarding-form/index.tsx
Comment (tataihono) [inline apps/journeys-admin/pages/onboarding-form/index.tsx:1]: you need to do `npm i` and rewrite this file. AuthUser was refactored to be just user in the latest version of next-firebase-auth.

**Review by tataihono — CHANGES_REQUESTED** [review]

File: libs/shared/ui/src/components/FormiumForm/Header/Header.tsx (hunk 2 above)
Comment (csiyang) [reply libs/shared/ui/src/components/FormiumForm/Header/Header.tsx:6]: Yes, I didn't intend the jsdoc comment

File: apps/journeys-admin/pages/onboarding-form/index.tsx
Comment (csiyang) [reply apps/journeys-admin/pages/onboarding-form/index.tsx:1]: nah, that page is going in as apart of a different PR, only had it in for testing purposes

**Review by tataihono — APPROVED** [review]

**Pull request merged**